### ISSUE TITLE
GUI version doesn't update the closure output

### ISSUE THREAD

**Issue by hrj** [hrj/abandon#63]

**Comment by gerdreiss** [comment]
Hi @hrj , I'm trying to pick the next task to take on, and this one seems to be appropriate, since it's planned for the "Next release" (together with #1, which seems to work though...). If you agree, I'd like to fix this bug, but need more info as to what exactly is the problem. Could you point me in the right direction? Thanks!

**Comment by hrj** [comment]
Hi @gerdreiss 

Actually I am not sure what this issue is about. I should have recorded the problem better.

My guess is that, I was surprised when exported transactions with closure were not being emitted after running GUI and I quickly jotted down the problem, without realizing that none of the *exports* are processed in the GUI. Only *reports* are shown.

TLDR: The abandon GUI is currently just a viewer, and not a processor. And I had forgotten that.
